assets/source/setup-guide/app/components/SaveSettingsButton.js

```diff
@@ -53,6 +56,26 @@ const SaveSettingsButton = ( { view } ) => {
 		try {
 			await setAppSettings( {} );
 
+			if (
+				appSettings?.tracking_advertiser &&
+				appSettings?.tracking_tag
+			) {
+				const result = await connectAdvertiser(
+					appSettings.tracking_advertiser,
+					appSettings.tracking_tag
+				);
+
+				if ( result.reconnected ) {
```

Comment:
🚧 `connectAdvertiser` now returns boolean true/false, there is no `reconnected` property anymore, so this `if` statement will always be evaluated to `false`.

I assume this should now be the following instead?

```suggestion
				if ( result ) {
```